### DISCUSSION TITLE
cj/1

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#874] [Announcements]
# cj/1

本节介绍 编写自己的第一个仓颉程序 第一个仓颉程序 上面这个程序的输出结果为： 通常，我们会在扩展名为 .cj 的文本文件中编写仓颉程序。 从上面这个程序中，我们可以看到，这个程序由main函数，println函数，以及

https://www.cjdev.cn/cj/1.html

**Comment by dapeng17951** [comment]
支持的操作系统有那些呢？window xp 支持吗？

**Reply by houyunlong1668** [reply]
应该可以doge

**Reply by dapeng17951** [reply]
cjc main.cj i386-windows-gnu   i386不支持呀

**Reply by houyunlong1668** [reply]
哦哦，我只是随便说说，主要是感觉xp很少见了，办公室最低也应该是win7，蹲一下大神来回答吧，我是菜鸟，这个网站挺高级自动发邮件，我都没反应过来。

**Reply by dapeng17951** [reply]
我也是菜鸟，工业上xp nt4.0 还是挺多的，vlang可以支持xp

**Reply by houyunlong1668** [reply]
问了一下群友说仓颉不支持xp,说仓颉使用的llvm15 xp要支持llvm3.8之前的版本

**Reply by dapeng17951** [reply]
谢谢解答，GOOD！
